nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server_impl.hpp

```diff
@@ -246,6 +271,32 @@ void BtActionServer<ActionT>::executeCallback()
   }
 }
 
+template<class ActionT>
+void BtActionServer<ActionT>::populateErrorCode(
+  typename std::shared_ptr<typename ActionT::Result> result)
+{
+  int highest_priority_error_code = std::numeric_limits<int>::max();
+  for (const auto & error_code_id_name : error_code_ids_) {
+    int current_error_code;
+    bool found_error_code = blackboard_->get<int>(error_code_id_name, current_error_code);
+
+    if (found_error_code) {
```

Comment:
```suggestion
    if (blackboard_->get<int>(error_code_id_name, current_error_code)) {
```

Then you can delete L281